shared/js/textChatView.jsx

```diff
@@ -261,6 +275,7 @@ loop.shared.views.chat = (function(mozL10n) {
                 return (
                   <TextChatEntry contentType={entry.contentType}
                                  dispatcher={this.props.dispatcher}
+                                 displayName={entry.displayName ? entry.displayName : undefined}
```

Comment:
I believe `entry.displayName` would be `undefined` in the case that you would have the tertiary result in `undefined` ..?